TombEngine/Game/effects/effects.cpp

```diff
@@ -1231,6 +1231,15 @@ void KillAllCurrentItems(short itemNumber)
 	// TODO: Reimplement this functionality.
 }
 
+void SpawnDynamicLight(const Vector3& pos, const Color& color, float falloff)
```

Comment:
Why this function is called `SpawnDynamicLight`, if we have a function called `TriggerDynamicLight` already? Make an overload.

Reply:
`SpawnDynamicLight()` is a replacement for `TriggerDynamicLight()` that uses the newer, consistently applied prefix convention decided upon for new or revised effect functions (`Spawn`).

Reply:
I think you can do en masse rename in other PR then. Having 2 functions with same purpose but named differently is misleading.